go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/alicebob/miniredis/v2 v2.17.0
 	github.com/dell/goisilon v1.9.0
-	github.com/dell/gopowermax/v2 v2.0.0
+	github.com/dell/gopowermax/v2 v2.0.1-0.20221109120540-a5532f923a52
 	github.com/dell/goscaleio v1.8.0
```

Comment:
Is there an update for goscaleio and goisilon?

Reply:
Last change made on goscaleio was in September with the latest release of 1.8.0, and goisilon was not included on the list of my story but checking right now it seems to be the case that both were last updated in September, so no commits on the respective go.mod files since or before that date.

Reply:
goisilon is gopowerscale. I do see a commit there from 5 days ago.

Reply:
checked again, you are right. Updating both goisilon and goscaleio to latest commit, was looking at the last commit on the go.mod files rather than the repo